File: Linear Search/linear.php
<?php 

function search($x , $arr)
{

    $n= sizeof($arr);

    for ($i=0; $i < $n ; $i++) { 
       if($arr[$i] == $x){
           return $i;
       }
       return -1;
    }
}

$arr = array(2, 3, 4, 10, 40);
$x = 10;

$result  =  search($x, $arr);

if ($result == -1){
    echo "Element is not present in arr";
}
else{
    echo "Element is present in arr";
}

?>

File: Linear Search/linear.py

def search(arr, n, x):
    for i in range(0, n):
        if(arr[i] == x):
            return i
    return -1


arr = [2, 3, 4, 5, 6, 7, 8]
x = 5
n = len(arr)

result = search(arr, n, x)
if(result == -1):
    print("Element is not present in an array.")
else:
    print("Element is present at index", result)
